src/i18n/de/email.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Hello Recogito Studio User": "Hello Recogito Studio User",
+  "has invited you to join the": "has invited you to join the",
+  "project on": "project on",
+  "Use the button below to accept the invitation, or accept the invitation from the Notifications button next time you log in.": "Use the button below to accept the invitation, or accept the invitation from the Notifications button next time you log in.",
+  "_you_have_been_invited_": "You have been invited to a project"
+}
```

Comment:
The word order doesn't work like that in German. We'd need a template, i.e. something like this:

```
{{sender}} has invited you to join the {{project_name}} project on {{host}}. Use the button to accept the invitation, or accept the invitation from the Notifications menu next time you log in.
```

In DE, the template would be:

```
{{sender}} hat dich in das Projekt {{project_name}} auf {{host}} eingeladen. Du kannst die Einladung mit Klick auf den Button akzeptieren, oder das nächste mal wenn Du Dich in Recogito einloggst über das Menü "Benachrichtigungen".
```

(Note the word after {{host}}.)